pages/om-abakus.tsx

```diff
@@ -137,7 +137,42 @@ const OmAbakus = () => {
       </AboutCard>
       <br />
 
-      <h3 className={styles.subTitle}>Komiteer</h3>
+      <AboutCard title="Hvordan engasjere seg?">
+        <b>Komiteene og revyen</b>
+        <p>
+          Opptak til{" "}
+          <Link href="https://abakus.no/pages/grupper/104-revyen">revyen</Link>{" "}
+          og <Link href="#comittees">komiteene</Link> skjer vanligvis i
+          september etter semesterstart.{" "}
+          <Link href="https://abakus.no/pages/komiteer/5">backup</Link> har
+          vanligvis opptak i februar. \n Opptaket gjennomføres på{" "}
+          <Link href="https://opptak.abakus.no">opptak.abakus.no</Link> og vil
+          annonseres på <Link href="https://abakus.no">abakus.no</Link> og i
+          sosiale medier.
+        </p>
+        <br />
+        <b>Undergrupper</b>
+        <p>
+          Undergruppene sine opptak varierer fra gruppe til gruppe. Se hver
+          enkelt undergruppe sin side på{" "}
+          <Link href="https://abakus.no">abakus.no</Link> for mer informasjon.
+        </p>
+        <br />
+        <b>Interessegrupper</b>
+        <p>
+          Interessegruppene i Abakus er åpne for alle. Man kan melde seg inn ved
+          å trykke på {'"'}Bli med i gruppen{'"'} på interessegruppens side på{" "}
```

Comment:
Is the {'"'} really needed?

Reply:
Hmm, I personally think so as it indicates that "Bli med i gruppen" is the button label

Reply:
Ah yeah I meant if you can't just type it as " in the text

Reply:
Ah well the linter didn't really like it, but I found a solution now using `&quot;` instead.